docs/docs/command-line-flags.md

```diff
@@ -41,6 +41,8 @@ The core functionality flags can be also set by environment variable `MARATHON_O
     Requires checkpointing enabled on slaves. Allows tasks to continue running
     during mesos-slave restarts and Marathon scheduler failover.  See the
     description of `--failover_timeout`.
+* <span class="label label-default">v0.12.0</span> `--no_checkpoint`
```

Comment:
still the "no" prefix.

I am not quite sure of the style. Do we list the "--disable_*" options as separate options or do we summarize it. E.g.

```
* `--[disable_]checkpoint` (Optional. Default: true): Enable checkpointing of tasks.
     Requires checkpointing enabled on slaves. Allows tasks to continue running
     during mesos-slave restarts and Marathon scheduler failover.  See the
     description of `--failover_timeout`.
```

One could also argue that you only need to document `--disable_checkpoint` because `--checkpoint` has no effect.